atmo/settings.py

```diff
@@ -394,6 +394,9 @@ class Core(AWS, Celery, Constance, CSP, Configuration):
                     'django.template.loaders.filesystem.Loader',
                     'django.template.loaders.app_directories.Loader',
                 ],
+                'libraries': {
+                    'atmo_tags': 'atmo.templatetags',
```

Comment:
Nice, I love that we don't have to put the template tags in submodules anymore. Could you rename the label though, so we can do `{% load atmo %}` in the templates instead the duplicated meaning `{% load atmo_tags %}`?